indigo/indigo/src/main/scala/indigo/shared/platform/DisplayObjectConversions.scala

```diff
@@ -172,66 +186,93 @@ final class DisplayObjectConversions(
   def sceneNodeToDisplayObject(
       gameTime: GameTime,
       assetMapping: AssetMapping,
-      cloneBlankDisplayObjects: => HashMap[CloneId, DisplayObject]
-  )(sceneNode: SceneGraphNode): DisplayEntity =
+      cloneBlankDisplayObjects: => HashMap[CloneId, DisplayObject],
+      renderingTechnology: RenderingTechnology,
+      maxBatchSize: Int
+  )(sceneNode: SceneGraphNode): (DisplayEntity, scalajs.js.Array[(CloneId, DisplayObject)]) =
```

Comment:
This is the ugliest bit, the return type of `(DisplayEntity, scalajs.js.Array[(CloneId, DisplayObject)])`.
Technically, that js array could be an option since there's only ever one. However, option is a special case of list/array, and all we're going to do is smoosh them all together, never test for their presence, nor do we need a null check. So js array should be the fastest collection type in this case.